app/src/main/java/io/homeassistant/companion/android/sensors/SensorDetailFragment.kt

```diff
@@ -147,6 +152,61 @@ class SensorDetailFragment(
             } else
                 it.isVisible = false
         }
+
+        findPreference<PreferenceCategory>("sensor_settings")?.let {
+            if (sensorData.enabled && !settings.isNullOrEmpty()) {
+                settings.forEach { setting ->
```

Comment:
Maybe we should sort these?  Not sure if I have a good suggestion on what to sort on though...

Reply:
So that was why I was thinking the manager should be responsible for the order they create the settings. This way when we create the page they appear in an expected order.  If we do any sorting then the managers would need to make sure the names are in expected order. Since the setting names are displayed to the user in a `key` format I felt this to be a better approach.